src/Microsoft.AspNet.Routing/ServiceCollectionExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static class ServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The services available in the application.</param>
         /// <param name="setupAction">An action to configure the <see cref="RouteOptions"/>.</param>
-        public static void ConfigureRouteOptions(
+        public static void ConfigureRoute(
```

Comment:
good point @kichalla, `ConfigureRoute` isn't correct because the `setupAction` can affect multiple routes